### ISSUE TITLE
Can't click when game window is not focused

### ISSUE THREAD

**Issue by memesnightmares** [ppy/osu-framework#4295]
**Describe the bug:** 

Since the OTD update, I've been unable to click with my tablet pen when the game window isn't focused. I'm unsure if this is a OTD issue or an osu issue. There are no errors in the log. Prior to updating to 2021.320, I did not use any tablet drivers.

**Screenshots or videos showing encountered issue:** Can't really show a video, but if I move my mouse outside of the osu window, I can still move my pen on my tablet to move the game cursor while my outside cursor stays static. However clicking with the tablet pen only registers when the game window has focus, which it doesn't when my mouse is outside.

**osu!lazer version:** 2021.320.0.0

**Logs:**

Unsure which logs to attach, as they all look fine and I have a lot of log files in my logs dir.

**Comment by memesnightmares** [comment]
This is actually a more general issue of focus handling, I think that the general solution / feature request here is for the game window to grab focus when it receives tablet input.

**Comment by frenzibyte** [comment]
The right way to solve this would likely just be to warp the mouse to the current position of the tablet input. (probably via `MouseHandler.FeedbackMousePosition(...)`)

**Comment by MrGhost7969** [comment]
Yeah, I've been having an issue like that earlier today.  I just purchased the Intuos pro (for sculpting and such) and wanted to see if it would work for osu, but then, the cursor was just stuck on to the top left edge of the screen and I couldn't move it with my stylus.

**Comment by bdach** [comment]
@MrGhost7969 you seem to be describing an entirely separate issue. this issue is about the game not handling *clicks via stylus* if the window isn't focused. it is not about the cursor just refusing to move at all and being stuck in a corner.
